### ISSUE TITLE
Modernize app user interface

### ISSUE THREAD

**Issue by hoffe86** [openhab/openhab-windows#131]
Modernize the application ui and ux.

- General 
   - [x] Use Acrylic effects 

- UX
   - [x] First Run experience (information about what needs to be done to configure a connection to a OpenHab server or cloud instance.

- Settings
   - [x] Rework settings dialog
   - [x] Add additional UI settings

- Adding multi language support
   - [x] German
   - [x] English

- Controls Improvements
   - [ ] Use Toggle-Switch for "switch items"
   - [ ] Use "normal" slider for "dimmer items"
   - [ ] Smaller icons inside widgets to get more space

- Widgets Improvements
   - [ ] AddaptiveGridView (windowscommunitytoolkit) for Widgets.

Additional ideas are welcome

**Comment by cd-tronic** [comment]
- use Toggle-Switch for "switch items" 
- remove/change Background-image? ("don't like it personally" and i think it's not fluent design language?`)
- use "normal" slider for "dimmer items"
- smaller icons inside widgets to get more space 
- addaptiveGridView (windowscommunitytoolkit) for Widgets.

Acrylic effect is nice!

**Comment by cd-tronic** [comment]
i will start developing with you as far as i got some more time. I'm pretty busy at the moment.

**Comment by hoffe86** [comment]
@cd-tronic Sounds fine for me
I have already prepared a branch with my latest UI changes [features/ui-modernization](https://github.com/openhab/openhab-windows/tree/features/ui-modernization)

**Comment by Cossey** [comment]
Would it be possible to Implement the `colorpicker` element? Used for changing smart light colours.

**Comment by hoffe86** [comment]
Hi @Cossey,
There is already a issue #73 for that.
I will add this feature to the v2020.1 milestone plan.

**Comment by sbaum2s** [comment]
Is this project still alive? I see pending pull requests and failed builds among other things. Looks abandoned.

**Comment by hoffe86** [comment]
Yes it is still alive, but in the last time I had no much time to work on the update.
My plan is to publish an update on the Beta App over the Christmas.

**Comment by sbaum2s** [comment]
Ok, i just wondered because the pull requests date back 1+ years @hoffe86
